packages/app/metrics-utils/package.json

```diff
@@ -0,0 +1,50 @@
+{
+    "name": "@lokalise/metrics-utils",
```

Comment:
can you also add readme, explaining how to use and what it does?

Reply:
I enhanced the readme with usage instructions. Could you please see if it's clearer now from your point of view?